encryption_works.md

```diff
@@ -83,7 +83,7 @@ Stories about the [lengths](http://www.nytimes.com/2013/08/18/magazine/laura-poi
 
 There are dozens of reasons people should use encryption even if you're not a journalist or a whistleblower—your medical details, credit card info, and personal photos shouldn't be fair game for hackers or governments.
 
-The NSA isn't the only adversary journalists have to be wary of. Imagine a newsroom that relies on a cloud-based storage system for documents. You and your colleagues are working on a story over the course of a few weeks. Just as you're beginning to edit the final draft, a reporter is fired. He returns his work laptop but nobody thinks to revoke his access to the cloud storage system your newsroom uses. He takes the story and the months of research to a rival newspaper and scoops you.
+The NSA isn't the only adversary journalists have to be wary of. Imagine a newsroom that relies on a cloud-based storage system for documents. You and your colleagues are working on a story over the course of a few weeks. Just as you're beginning to edit the final draft, a reporter is fired. They return their work laptop but nobody thinks to revoke their access to the cloud storage system your newsroom uses. They take the story and the months of research to a rival newspaper and scoops you.
 
```

Comment:
They take the story and the months of research to a rival newspaper and _scoop_ you.